app/components/profile/terms-of-use/TermsOfUseForm.js

```diff
@@ -27,7 +29,7 @@ type Props = {
   onSubmit: Function,
   isSubmitting: boolean,
   error?: ?LocalizableError,
-  classicTheme: boolean
+  currentTheme: Theme
```

Comment:
I think you could also pass the function `isClassic`

Reply:
@nicarq 
yes, we are thinking to change it in one go at all the places.
As change will affect more than 50 files so thinking to make change in different PR.
cc @YankoFelipe

Reply:
sounds good! let’s not continue adding more of them in other PRs though 😄

Reply:
@nicarq 
Yes, we are following the approach to remove `classicTheme: boolean` but in some exceptional case we have to keep it, will think in deep if we can remove it completely.
cc @YankoFelipe